jobs/taar_lite_guidguid.py

```diff
@@ -94,18 +77,47 @@ def get_addons_per_client(broadcast_amo_whitelist, users_df):
     return df
 
 
+def get_table(view):
+    """Helper for determining what table underlies a user-facing view, since the Storage API can't read views."""
+    bq = bigquery.Client()
+    view = view.replace(":", ".")
+    # partition filter is required, so try a couple options
+    for partition_column in ["DATE(submission_timestamp)", "submission_date"]:
+        try:
+            job = bq.query(
+                f"SELECT * FROM `{view}` WHERE {partition_column} = CURRENT_DATE",
+                bigquery.QueryJobConfig(dry_run=True),
+            )
+            break
+        except Exception:
+            continue
+    else:
```

Comment:
else block doesn't match try/except indentation

Reply:
good catch, I'll fix this. we probably should report to telemetry docs owners

Reply:
actually, this is `for .. else` construction which I see the first time myself:) https://book.pythontips.com/en/latest/for_-_else.html#else-clause That's how they handle errors here.

Reply:
ah, cool